### PR TITLE
Add RuboCop offense on Dangerfile

### PR DESCRIPTION
**Pull request by iangmaia**
Test Pull Request

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
```
Comment (dangermattic) [inline Dangerfile:1]: <table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/iangmaia/danger-rubocop-test/blob/edc50e8dd0680ef25054451a7e63782fe6b006ce/Dangerfile#L1"></span>Style/FrozenStringLiteralComment: Missing frozen string literal comment.</td>
    </tr>
  </tbody>
</table>

File: Dangerfile [hunk 2]
```diff
@@ -1,4 +1,8 @@
 
 markdown("PR number: #{github.pr_json['number']}")
```
Comment (dangermattic) [inline Dangerfile:2]: <table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/iangmaia/danger-rubocop-test/blob/edc50e8dd0680ef25054451a7e63782fe6b006ce/Dangerfile#L2"></span>Layout/LeadingEmptyLines: Unnecessary blank line at the beginning of the source.</td>
    </tr>
  </tbody>
</table>

File: Dangerfile [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+markdown("PR number: #{github.pr_json['mergeable']}")
```
Comment (dangermattic) [inline Dangerfile:3]: <table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/iangmaia/danger-rubocop-test/blob/edc50e8dd0680ef25054451a7e63782fe6b006ce/Dangerfile#L3"></span>Layout/TrailingWhitespace: Trailing whitespace detected.</td>
    </tr>
  </tbody>
</table>

File: Dangerfile [hunk 4]
```diff
@@ -1,4 +1,8 @@
 
 markdown("PR number: #{github.pr_json['number']}")
+markdown("PR number: #{github.pr_json['mergeable']}") 
+
+rubocop.lint(files: [],
+ force_exclusion: true, inline_comment: true, report_severity: true, include_cop_names: true)
```
Comment (dangermattic) [inline Dangerfile:6]: <table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/iangmaia/danger-rubocop-test/blob/edc50e8dd0680ef25054451a7e63782fe6b006ce/Dangerfile#L6"></span>Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.</td>
    </tr>
  </tbody>
</table>

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/iangmaia/danger-rubocop-test/blob/edc50e8dd0680ef25054451a7e63782fe6b006ce/Gemfile#L3">Gemfile#L3</a> - Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>

PR number: 1

PR number: true

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by iangmaia** [comment]
PR number: 1

PR number: true

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Dangerfile [hunk 5]
```diff
@@ -1,4 +1,7 @@
+# frozen_string_literal: true
 
-markdown("PR number: #{github.pr_json['number']}")
+markdown("PR number: #{github.pr_json['number']}") 
+markdown("PR number: #{github.pr_json['mergeable']}")
+
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true,  report_severity: true, include_cop_names: true)
```
Comment (iangmaia) [inline Dangerfile:7]: <table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/iangmaia/danger-rubocop-test/blob/85e3a876703db7fa68f2c2c0d4b315b4c61e0342/Dangerfile#L6"></span>Layout/ExtraSpacing: Unnecessary spacing detected.</td>
    </tr>
  </tbody>
</table>

**Comment by iangmaia** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/iangmaia/danger-rubocop-test/blob/85e3a876703db7fa68f2c2c0d4b315b4c61e0342/Dangerfile#L7">Dangerfile#L7</a> - Layout/TrailingEmptyLines: 1 trailing blank lines detected.</td>
    </tr>
  </tbody>
</table>

PR number: 1

PR number: true

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by iangmaia** [comment]
PR number: 1

PR number: true

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request closed without merging**